scheduler/src/cook/tools.clj

```diff
@@ -28,6 +28,7 @@
             [cook.cached-queries :as cached-queries]
             [cook.caches :as caches]
             [cook.config :as config]
+            [cook.datomic :as datomic]
```

Comment:
Why was this needed?

Reply:
Sorry that's my fault, I added the requirement for the cache functions but forgot to take it out after removing them. I'll remove it in the next revision.